base/lnls-prune-artifacts.sh

```diff
@@ -106,5 +112,23 @@ remove_static_libs() {
     done
 }
 
+remove_unused_shared_libs() {
+    target_libs=$(find_shared_libs $@)
+    linked_libs=$(find_linked_libraries $@)
+    remove_libs=$(find_shared_libs /opt /usr/local)
```

Comment:
Adding `/usr/local` here is due to #55, right? Might make sense to mention in the commit that it's future proofing, and doesn't actually change behavior for now.